src/CoreWCF.Primitives/src/CoreWCF/Channels/ByteStreamMessageUtility.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Xml;
+using CoreWCF.Runtime;
+
+namespace CoreWCF.Channels
+{
+    internal static class ByteStreamMessageUtility
+    {
+        public const string StreamElementName = "Binary";
+        public const string XmlNamespace = "http://www.w3.org/XML/1998/namespace";
+        public const string XmlNamespaceNamespace = "http://www.w3.org/2000/xmlns/";
+
+        // used when doing message tracing
+        internal const string EncoderName = "ByteStreamMessageEncoder";
+
+        internal static void EnsureByteBoundaries(byte[] buffer, int index, int count, bool isRead)
+        {
+            if (buffer == null)
+            {
+                throw Fx.Exception.ArgumentNull("buffer");
+            }
```

Comment:
nameof for the validation exception field names

Reply:
This is addressed in b74c8b433b881ab07669c472c7a52b24518baf27. I looked for other places that an argument related exception was being raised and used nameof there as well.